app/routes/circulars/circulars.lib.ts

```diff
@@ -130,6 +132,15 @@ export function formatIsValid(format: string): format is CircularFormat {
   return (circularFormats as any as string[]).includes(format)
 }
 
+/** For updated dates, check that the date is valid */
+export function dateIsValid(date?: string, time?: string) {
+  return !Number.isNaN(Date.parse(`${date} ${time} UTC`))
```

Comment:
This doesn't work in my browser (Safari on macOS), and it prevents the submit button from ever being enabled.

From my browser's JavaScript console:

```
> Date.parse('2023-09-08 15:59 UTC')
NaN
```

Reply:
Does the following parse correctly on safari? 
```js
Date.parse('2023-09-08T15:59:00.000Z')
```

Based it off https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

Reply:
Yes.

Reply:
Okay i will adjust the formatting to something that will parse correctly